src/azure-cli-core/azure/cli/core/profiles/_shared.py

```diff
@@ -210,6 +210,45 @@ def default_api_version(self):
         ResourceType.MGMT_IOTHUB: '2020-03-01',
         ResourceType.MGMT_ARO: '2020-04-30'
     },
+    '2020-09-01-hybrid': {
+        ResourceType.MGMT_STORAGE: '2019-06-01',
+        ResourceType.MGMT_NETWORK: '2018-11-01',
```

Comment:
Just out of curiosity, Does the network API need so old? Why not prefer 2019 or newer

Reply:
AzureStack doesn't support the latest API version which are available on azure cloud.